remediation/fence-agents-remediation/limitations.md

```diff
@@ -10,3 +10,4 @@ nav_order: 3
 # Limitations
 
 1. Currently, FAR supports running a fence agent with only the *reboot* [action](https://github.com/ClusterLabs/fence-agents/blob/main/doc/FenceAgentAPI.md#agent-operations-and-return-values).
+2. The running fence agent inside the FAR pod needs network connectivity to reach the unhealthy node interface, and fence it. In some platforms, the pod isn't granted with this kind of permission.
```

Comment:
suggestions:

> The ~running~ fence agent in~side~ the FAR pod...

> ...needs network connectivity to ~reach~ the unhealthy node **'s management** interface ~, and fence it~.

this isn't a single  pod's permission, so maybe:

> ~In~ **On** some platforms, ~the pod isn't granted with this kind of permission.~ **the pod network  lacks this kind of connectivity.**

Reply:
> this isn't a single pod's permission, 

I forgot about the second replica